crates/typst/src/model/emph.rs

```diff
@@ -22,13 +22,13 @@ use crate::text::{ItalicToggle, TextElem};
 /// This is _emphasized_ differently.
 /// ```
 ///
-/// # Syntax
-/// This function also has dedicated syntax: To emphasize content, simply
-/// enclose it in underscores (`_`). Note that this only works at word
-/// boundaries. To emphasize part of a word, you have to use the function.
+/// # 構文
+/// この関数には専用の構文もあります。
+/// コンテンツを強調するには、単にアンダースコア(`_`)で囲みます。ただし、これは単語の区切りにおいてのみ機能します。
```

Comment:
他のページとの整合性を取るために、かっこを全角にしていただけますと幸いです。

```suggestion
/// コンテンツを強調するには、単にアンダースコア（`_`）で囲みます。ただし、これは単語の区切りにおいてのみ機能します。
```

Reply:
ありがとうございます。反映を行いました。